src/NBomber.YCSB/YcsbSettings.cs

```diff
@@ -0,0 +1,41 @@
+﻿using CommandLine;
+
+namespace NBomber.YCSB
+{
+    [Verb("run", HelpText = "Run a workload")]
+    public class YcsbSettings
```

Comment:
I would rename this to
YcsbCliArgs

also it make sense to move this file to Infra folder